feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -205,17 +242,13 @@ fun ForYouScreen(
 
                 newsFeed(
                     feedState = feedState,
-                    // Avoid showing a second loading wheel if we already are for the interests
-                    // selection
-                    showLoadingUIIfLoading =
-                    interestsSelectionState !is ForYouInterestsSelectionUiState.Loading,
                     onNewsResourcesCheckedChanged = onNewsResourcesCheckedChanged,
-                    loadingContentDescription = R.string.for_you_loading
                 )
 
                 item(span = { GridItemSpan(maxLineSpan) }) {
                     Column {
                         Spacer(modifier = Modifier.height(8.dp))
+                        if (isOffline) Spacer(modifier = Modifier.height(48.dp))
```

Comment:
Just confirming, this is to add extra space for the snackbar?

Reply:
Yeah, too magic number-y?

Reply:
A little bit, but I don't really see a better solution. Maybe just add a comment saying what there's extra space?